tests/kafkatest/services/streams.py

```diff
@@ -45,11 +45,14 @@ class StreamsTestBaseService(KafkaPathResolverMixin, Service):
             "collect_default": True},
     }
 
-    def __init__(self, test_context, kafka, streams_class_name, user_test_args):
+    def __init__(self, test_context, kafka, streams_class_name, user_test_args, user_test_args1, user_test_args2):
         super(StreamsTestBaseService, self).__init__(test_context, 1)
         self.kafka = kafka
         self.args = {'streams_class_name': streams_class_name,
-                     'user_test_args': user_test_args}
+                     'user_test_args': user_test_args,
+                     'user_test_args1': user_test_args1,
```

Comment:
I am not against it.

Just wondering, if it might work to pass a single user-test-arg, and if multiple args are needed, use a test specific formatting (like parameters separated with special char) and do a `String#split()` within the Java test code to get the individual parameters?

Both ways have advantages and disadvantages. Just want to put an alternative (not saying the alternative is better).